handbook/operations.md

```diff
@@ -0,0 +1,63 @@
+
+3. Share the compensation information you've gathered with Eric. Eric will confirm all this information with Mike and document this decision in the [compensation decisions document](https://docs.google.com/document/d/1NQ-IjcOTbyFluCWqsFLMfP4SvnopoXDcX0civ-STS5c/edit) for future reference.
+
+4. After compensation has been determined, you need to create two documents for the candidate, an "exit scenarios" spreadsheet and an "informal offer email" template. If you need these templates, reach out to Eric on slack, and he will provide you links that automatically create copies Change the name of the documents accordingly (e.g., "[candidate's name]'s copy of exit scenarios,") and link to the exit scenarios spreadsheet from the offer email.
+
+5. Prepare the informal offer email. You'll need to fill the following information to the template: 
+   - Candidate's name and email address
+   - Candidate's start date
+   - Candidate's compensation
+   - Candidate's reporting manager
+   - Equity offered to the candidate (make this information a link to the candidate's exit scenarios spreadsheet)
+   - Benefits (determined by the candidate's location)
+
+6. Prepare the exit scenarios spreadsheet. Enter the percentage of equity offered to the candidate, and the spreadsheet will update to reflect this. 
+   
+      >**_Note:_** *Don't play with numbers in the exit scenarios spreadsheet. the revision history is visible to the candidate, and they might misunderstand.*
+
+7. Once both documents are complete, share the offer email draft, exit scenarios copy, and a link to the compensation decision, and send them to Eric for confirmation.
+
+8. After Eric confirms that everything is correct, Mike or Zach will send the offer email. The offer email is copied directly from Google drive and sent to the candidate. When they send the offer, Mike or Zach will edit the permissions of the exit scenarios sheet and share with the candidate.
+
+   >**_Note:_** *When hiring an international employee, Pilot.co recommends starting the hiring process a month before the new employee's start date.*
+
+### Steps after an offer is accepted
+
+1. Once an offer is accepted in writing, reply to the candidate, CCing Eric via his Fleet email address to introduce the candidate to them.
+
+2. If the new team member is in the United States, Eric will reach out to them and get any information needed (typically the new team member's home address and phone number) and send them a consulting or employment agreement through [Docusign](https://www.docusign.com/). If the new team member is an international employee or contractor, Eric will start the hiring process in [Pilot](https://pilot.co/).
+
+   >**_Note:_** *International contractor and employment agreements are handled by the Pilot.co when you start the hiring process.*
+
+3. **For US employees/contractors** After an agreement is signed and stored in the correct Google
```

Comment:
Is there a reason why all of point 3 has been split over separate lines? I assume it's a copy/paste error or something?